org.eclipse.xtext.ide.tests/src/org/eclipse/xtext/ide/tests/serializer/ChangeSerializerTestHelper.xtend

```diff
@@ -40,17 +40,17 @@ class ChangeSerializerTestHelper {
 
 	def void ===(Collection<IEmfResourceChange> actual, CharSequence expected) {
 		val actualString = new TextDocumentChangeToString().add(actual).toString
-		Assert.assertEquals(expected.toString.trim, actualString.trim)
+		Assert.assertEquals(expected.toString.trim.replaceAll("\r\n","\n"), actualString.trim.replaceAll("\r\n","\n"))
```

Comment:
Please use this [`Strings#toPlatformLineSeparator `](https://github.com/eclipse/xtext-core/blob/master/org.eclipse.xtext.util/src/org/eclipse/xtext/util/Strings.java#L105) instead.

Reply:
but i want \n not the win vs unix stuff Strings#toPlatformLineSeparator does